test/integration/tasks.test.js

```diff
@@ -371,13 +392,47 @@ describe("Tasks", function () {
     });
 
     it("Should give 403 if status is already 'VERIFIED' ", async function () {
+      taskId = (await tasks.updateTask({ ...taskData, assignee: appOwner.username })).taskId;
+      const res = await chai
+        .request(app)
+        .patch(`/tasks/self/${taskId}`)
+        .set("cookie", `${cookieName}=${jwt}`)
+        .send(taskStatusData);
+
+      expect(res).to.have.status(403);
+      expect(res.body.message).to.be.equal("Status cannot be updated. Please contact admin.");
+    });
+    it("Should give 403 if new status is 'MERGED' ", async function () {
```

Comment:
An HTTP 403 response code means that a client is forbidden from accessing a valid URL

Does status 403 hold good here? if yes can you explain

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403

Reply:
yes, it is good here as I'm trying to forbid the user to put the status as merged 
I want to give that functionality only to the super-user only